src/main/java/com/sonyericsson/jenkins/plugins/bfa/PluginImpl.java

```diff
@@ -504,6 +512,7 @@ public KnowledgeBase getKnowledgeBase() {
     public void configure(StaplerRequest req, JSONObject o) throws Descriptor.FormException, IOException {
         noCausesMessage = o.getString("noCausesMessage");
         globalEnabled = o.getBoolean("globalEnabled");
+        doNotAnalyzeAbortedJob = getBooleanWithDefault(o, "doNotAnalyzeAbortedJob", Boolean.FALSE);
```

Comment:
you don't need a `Boolean` object here since you are never using the `null` value.

And for that you have `o.optBoolean(key, defaultValue)`